examples/Huggingface_Transformers/Image_classification_docker/HF-only/README.md

```diff
@@ -0,0 +1,158 @@
+
+
+# HF only
+
+## Setup
+Log into the terminal of an AWS EC2 `t2.micro` instance with a base AWS Deep Learning PyTorch AMI (as it includes [`torch-model-archiver`](https://github.com/pytorch/serve/tree/master/model-archiver) and `docker`)
+
+The `Using Torchserve Docker containers on EC2 for inference` section of [this Notion blog](https://www.notion.so/Day-2-3-Torchserve-custom-handlers-and-Docker-containers-02665de910a64aedab2b907a9a0cc9b0#3825849dfb8942379df2cdce8a729d9a) by me can be followed for logging into the appropriate AWS EC2 instance.
```

Comment:
I'm getting a "you do not have access error" -  I think we should ideally keep the new docs self contained

Reply:
Hi @msaroufim ,
I am also excited to work on this and it could indeed make the code for HF & TS integration more compact.

(i) I think it makes sense that we should integrate the whole thing into the TS repo itself, once the PR is final, I would update my repo to directly import dependencies from `serve/examples/Huggingface_Transformers` itself and continue working on integrating inference-time optimizations from [`optimum`](https://github.com/huggingface/optimum) library into my repo (which could also be a future PR here). 
(ii) It is indeed possible to generalize my current PR to all the supported tasks with some more functions, if-else statements.
(iii) That makes sense, I would try to integrate `torchserve_vitxxsmall_handler.py` completely into `Transformer_handler_generalized.py` and remove `Transformer_handler_generalized.py` altogether from the new `Image_classification_docker` folder altogether.

The link seems to be faulty, it was supposed to be [this](https://garrulous-saxophone-8a6.notion.site/Day-2-3-Torchserve-custom-handlers-and-Docker-containers-02665de910a64aedab2b907a9a0cc9b0), which is basically supposed to be a foolproof guide to launch the EC2 instance by those who have never used AWS before (the same was the case with me till a few days ago so I decided to log it).

For maintaining the formality of the documentation, we could remove the reference to my journal altogether from the PR.